charts/app-store/templates/backend/backend.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: appstore-backend
+  labels:
+    app: appstore-backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: appstore-backend
+  template:
+    metadata:
+      labels:
+        app: appstore-backend
+      annotations:
+        checksum/config: "{{ include (print $.Template.BasePath "/backend/appstore-backend-config.yaml") . | sha256sum }}"
+    spec:
+      containers:
+      - name: appstore-backend
+        image: "{{ .Values.appstore-backend.image }}"
+        ports:
+        - containerPort: {{ .Values.appstore_backend.port }}
```

Comment:
in the `values.yaml` file shouldn't `appstore-backend` be just `backend` to match the value?
the same goes for `containerPort` which should be `backend`

Reply:
done in values